x-pack/plugins/security_solution/cypress/integration/host_details/risk_tab.spec.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { loginAndWaitForHostDetailsPage } from '../../tasks/login';
+
+import { cleanKibana } from '../../tasks/common';
+import { esArchiverLoad, esArchiverUnload } from '../../tasks/es_archiver';
+import { TABLE_CELL, TABLE_ROWS } from '../../screens/alerts_details';
+import {
+  navigateToHostRiskDetailTab,
+  openRiskFlyout,
+  waitForTableToLoad,
+} from '../../tasks/host_risk';
+
+describe('risk tab', () => {
+  before(() => {
+    cleanKibana();
+    esArchiverLoad('risky_hosts');
+  });
+
+  after(() => {
+    esArchiverUnload('risky_hosts');
+  });
+
+  it('renders risk tab', () => {
+    loginAndWaitForHostDetailsPage('siem-kibana');
```

Comment:
As we are not doing any change to the data, you can move the: `loginAndWaitForHostDetailsPage`, `navigateToHostRiskDetailTab`, and `waitForTableToLoad` methods into the before hook for both tests in this file. In this way, we improve the execution time of the test (checking in my local we improve from 63,79 to 39,36).